R/Rules-class.R

```diff
@@ -915,58 +915,58 @@ IncrementsRelativeDLTCurrent <- function(dlt_intervals = c(0, 1),
   )
 }
 
-# IncrementsNumDoseLevels ----
+# IncrementsDoseLevels ----
 
 ## class ----
 
-#' `IncrementsNumDoseLevels`
+#' `IncrementsDoseLevels`
 #'
 #' @description `r lifecycle::badge("experimental")`
 #'
-#' [`IncrementsNumDoseLevels`] is the class for increments control based on the
-#' number of maximum dose levels to increment.
+#' [`IncrementsDoseLevels`] is the class for increments control based on the
```

Comment:
```suggestion
#' [`IncrementsDoseLevels`] is the class for increments control based on the number of
```
still important to talk about number of dose levels. This is the main difference of this function compared e.g. to dose proportion increment